libs/server/Objects/Hash/HashObjectImpl.cs

```diff
@@ -148,6 +148,69 @@ private void HashLength(byte* output)
             ((ObjectOutputHeader*)output)->countDone = hash.Count;
         }
 
+        private void HashStrLength(byte* input, int length, ref SpanByteAndMemory output)
+        {
+            var _input = (ObjectInputHeader*)input;
+            int count = _input->count;
```

Comment:
Won't count always be 1 here? HSTRLEN doesn't accept more than 1 key per command

Reply:
You are right. I have rewritten the code to simplify the overhead needed to process.